todo_app/templates/index.html

```diff
@@ -4,15 +4,83 @@
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">This is not just any to-do app...</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
+      <ul id="todo_list" class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li id="li{{ item.id}}"><span>{{ item.title }} - </span><span id="status{{ item.id}})" onclick="updateStatus({{ item.id }}, event)">{{ item.status }}</span> <span onclick="deleteItem({{ item.id }})">(Delete)</span></li>
+        {% endfor %}
+        <li id="new_item" ><form id="new_todo_form" action="/todo/add" method="post" onsubmit="createNewItem(event)")>
+        <label for="title">New Todo:</label> <input type="text" id="title" name="title" value="" /> <input type="submit" value="Submit" />
+      </form></li>
       </ul>
     </div>
+
   </div>
+
+  <script>
+    function createNewItem(event) {
```

Comment:
Nice use of fetch & JS to avoid having to refresh the page all the time!

Reply:
Thanks 🙂